pandas/tests/arithmetic/test_period.py

```diff
@@ -1085,3 +1085,21 @@ def test_pi_sub_period_nat(self):
         exp = pd.TimedeltaIndex([np.nan, np.nan, np.nan, np.nan], name='idx')
         tm.assert_index_equal(idx - pd.Period('NaT', freq='M'), exp)
         tm.assert_index_equal(pd.Period('NaT', freq='M') - idx, exp)
+
+
+class TestPeriodArithmetic(object):
```

Comment:
This all belongs in pandas.tests.scalar.period.  If you want to make a new file test_arithmetic.py in that directory, that'd be OK.  Otherwise put in test_period.py

Reply:
moved